src/index.ts

```diff
@@ -136,12 +140,15 @@ class MongooseTsgen extends Command {
         this.log(`Writing interfaces to ${genFilePath}`);
 
         generator.saveFile({ genFilePath, sourceFile });
+
         if (!flags["no-format"]) await formatter.format([genFilePath]);
         this.log("Writing complete 🐒");
         process.exit();
       }
     } catch (error) {
-      this.error(error as Error);
+      const err = error as Error;
+      this.log(`❌ - ${err.stack}`);
```

Comment:
This should only be logged in debug mode, eg:

```
if (flags.debug) {
  this.log(err.stack)
}
```

Reply:
the -d option is not working, I had to do a debugger using the vscode tool to get to the error